events_protocol/core/logging/__init__.py

```diff
@@ -51,19 +51,23 @@ def __init__(self, klass=None):
     def log(self, level, msg, *args, **kwargs):
         if self.isEnabledFor(level):
             event_context = EventContextHolder.get()
-            _msg = dict(
-                severity=logging.getLevelName(level),
-                logmessage=msg,
+            event_payload = dict(
```

Comment:
```suggestion
            event_info = dict(
```
Só uma última chatice, bora mudar esse nome, porque `payload` tem um significado especial dentro da sintaxe do protocolo de eventos.